annotation-client/src/main/java/eu/europeana/annotation/client/connection/AnnotationApiConnection.java

```diff
@@ -242,7 +243,13 @@ public AnnotationOperationResponse createEuropeanaAnnotationLd(String motivation
 	/**
 	 * Execute Europeana API request
 	 */
-	return postURL(url, annotation, authorizationHeaderName, regularUserAuthorizationValue);
+	if (user!=null && user.equals(USER_ADMIN)) {
```

Comment:
this will not work when we use a third user. please create a method getAuthorizationHeaderValue(String user), and use it everywhere where needed. If the user is null, it should return the Header value for the regular user

Reply:
Done.